components/browser/session/src/main/java/mozilla/components/browser/session/tab/CustomTabConfig.kt

```diff
@@ -97,18 +97,22 @@ class CustomTabConfig internal constructor(
                 null
             }
 
-            val closeButtonIcon = run {
-                val icon = intent.getParcelableExtra(EXTRA_CLOSE_BUTTON_ICON) as? Bitmap
-                val density = displayMetrics?.density ?: 1f
-                if (icon != null &&
-                    icon.width / density <= MAX_CLOSE_BUTTON_SIZE_DP &&
-                    icon.height / density <= MAX_CLOSE_BUTTON_SIZE_DP
-                ) {
+            val closeButtonIcon = if (intent.hasExtra(EXTRA_CLOSE_BUTTON_ICON)) {
+                val bitmap = intent.getParcelableExtra(EXTRA_CLOSE_BUTTON_ICON) as? Bitmap
+                bitmap?.let { icon ->
+                    val density = displayMetrics?.density ?: 1f
+                    if (icon.width / density <= MAX_CLOSE_BUTTON_SIZE_DP &&
+                        icon.height / density <= MAX_CLOSE_BUTTON_SIZE_DP
+                    ) {
+                        icon
+                    } else {
+                        null
+                    }
+                }?.also {
```

Comment:
This is cleanup for a test that didn't specify a close icon, and showing a non-failing stacktrace.